docs/environment

```diff
@@ -1,28 +1,61 @@
-# Example environment file
+##
+## Example ~/.config/labwc/environment file.
+## Uncomment lines starting with one '#' to suit your needs.
+##
 
-# This allows xdg-desktop-portal-wlr to function (e.g. for screen-recording)
-XDG_CURRENT_DESKTOP=wlroots
+##
+## Use the XKB_DEFAULT_LAYOUT variable to set the keyboard layout. For example
+## to start with Swedish keyboard layout set it to 'se'. If you are unsure what
+## your country code is, refer to the layout section of:
+## /usr/share/X11/xkb/rules/evdev.lst
+##
+## Multiple keyboard layouts can be set by comma-separating the country codes.
+## If multiple layouts are used, specify the toggle-keybind using
+## XKB_DEFAULT_OPTIONS as show below.
+##
+## For further details, see xkeyboard-config(7)
+##
 
-# Set keyboard layout to Swedish
-XKB_DEFAULT_LAYOUT=se
+# XKB_DEFAULT_LAYOUT=se
+# XKB_DEFAULT_LAYOUT=se,de
+# XKB_DEFAULT_OPTIONS=grp:alt_shift_toggle
+# XKB_DEFAULT_OPTIONS=grp:shift_caps_toggle
 
-# Set two keyboard layouts and toggle between them using alt+shift
-XKB_DEFAULT_LAYOUT=se,de
-XKB_DEFAULT_OPTIONS=grp:alt_shift_toggle
+##
+## Force firefox to use wayland backend.
+##
 
-# Force firefox to use wayland backend
-MOZ_ENABLE_WAYLAND=1
+# MOZ_ENABLE_WAYLAND=1
 
-# Set cursor theme.
-# Find icons themes with the command below or similar:
-#     find /usr/share/icons/ -type d -name "cursors"
-XCURSOR_THEME=breeze_cursors
+##
+## Set cursor theme and size. Find system icons themes with:
+## `find /usr/share/icons/ -type d -name "cursors"`
+##
 
-# Disable hardware cursors. Most users wouldn't want to do this, but if you
-# are experiencing issues with disappearing cursors, this might fix it.
-WLR_NO_HARDWARE_CURSORS=1
+# XCURSOR_THEME=breeze_cursors
+# XCURSOR_THEME=capitaine-cursors
+# XCURSOR_SIZE=24
+
+##
+## Disable hardware cursors. Most users would not want to do this, but if you
+## are experiencing issues with disappearing cursors, this might fix it.
+##
+
+# WLR_NO_HARDWARE_CURSORS=1
+
+##
+## For Java applications such as JetBrains/Intellij Idea, set this variable
+## to avoid menus with incorrect offset and blank windows.
+## See https://github.com/swaywm/sway/issues/595
+##
+
+# _JAVA_AWT_WM_NONREPARENTING=1
```

Comment:
I don't think that this makes anything worse so I'd vote for leaving this active.

Reply:
I'd argue that having the example config file differ from the default config is still going to cause confusion. If we are going to recommend this in the example, maybe we should just enable it by default?

Reply:
Hm.. what about we set it ourselves before loading the environment file, stating that in the comment and changing it to `=0` int he example file.

The comment could then read something like
```
To avoid menus with incorrect offsets and blank windows with some Java
applications like JetBrains / Intellij Idea or JDownloader  we set the following
to `1` internally. It can be overwritten by uncommenting the following line.
```

(Assuming that JDownloader has the same issue and fix).

Reply:
Could work. Not a big deal to me, anyway.

Reply:
Have gone for setting this in `src/config/session.c` and adjusting `docs/environment` as @Consolatis suggested (ish).